index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <script src="https://kit.fontawesome.com/07d2052d30.js" crossorigin="anonymous"></script>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
```

Comment:
Great work using google fonts.
However,try and choose **only the fonts** and weight that **you need**. Importing them all cost more in terms of network usage thus  slower page load.

Reply:
Alright, thank you.

Reply:
Great work on the changes so far.
For the fonts when you go to select on Google Fonts you can choose the weights that you actually need instead of importing them all.

Example :
![Screenshot_2024-01-03_21_17_00](https://github.com/ClementCrown/Liprem/assets/16876681/fbd3fc50-261b-411a-9bbe-6f89c98a12ff)

Which gives me at the end 
**Roboto:ital,wght@0,400;0,900;1,400**&

Reply:
Okay